collector/scripts/sitemap.py

```diff
@@ -0,0 +1,26 @@
+import requests
+import xml.etree.ElementTree as ET
+from scripts.link import parse_links
+
+def parse_sitemap(url):
+    response = requests.get(url)
+    root = ET.fromstring(response.content)
+    
+    urls = []
+    for element in root.iter('{http://www.sitemaps.org/schemas/sitemap/0.9}url'):
+        for loc in element.iter('{http://www.sitemaps.org/schemas/sitemap/0.9}loc'):
+            urls.append(loc.text)
+    
+    return urls
+
+def sitemap():

```

Comment:
Can you add a comment above this line like there is in `links.py` to a URL of a sitemap (can be anywhere) as it helps people know what a good example target looks like.

Reply:
Decent example: `https://www.nerdwallet.com/blog/wp-sitemap-news-articles-1.xml`

Otherwise, looks good to merge